Modules/Scripted/DICOMLib/DICOMProcesses.py

```diff
@@ -348,13 +348,14 @@ class DICOMSender(DICOMProcess):
   (Uses storescu from dcmtk)
   """
 
-  def __init__(self,files,address,protocol=None,progressCallback=None):
+  def __init__(self,files,address,aeTitle,protocol=None,progressCallback=None):
```

Comment:
I've just noticed that this change would break those extensions that use DICOMSender class. An easy change would be to add a default None value to aeTitle (and it that case use the default "CTK" value), and maybe move it to the end of the argument list.

Reply:
Sure I do it